models/comment-models.js

```diff
@@ -29,3 +29,16 @@ exports.createNewComment = (comment, id) => {
       return rows[0];
     });
 };
+
+exports.removeCommentById = (id) => {
+  return db
+    .query("SELECT * FROM comments where comment_id = $1", [id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ msg: "Comment not found" });
+      }
```

Comment:
Valid way to check for the comment existing, but can be done more easily! You can streamline the function and not include this first query by adding a RETURNING * to the deletion query to get the rows that were removed, and if nothing comes back you can infer that the comment didn't exist.